build/Dockerfile

```diff
@@ -24,20 +24,16 @@ ENV STEAM_USER="steam" \
 	STEAM_HOME="/opt/steam"
 
 # Add steam user.
-RUN adduser --disabled-login --gecos "Steam" --home $STEAM_HOME $STEAM_USER
+RUN adduser --system --disabled-login --gecos "Steam" --home $STEAM_HOME $STEAM_USER
```

Comment:
> A home directory is created by the same rules as for normal users.  The	new  system  user will  have  the	shell  /bin/false  (unless  overridden with the --shell option), and have logins disabled.  **Skeletal configuration files are not copied.**

Extracted from [`man adduser`](http://www.unix.com/man-page/Linux/8/adduser/). The emphasys is mine.

It does other things too but I don't see any wrong with them. You can read the *Add a system user* section.